src/main/java/fr/lunatech/timekeeper/models/User.java

```diff
@@ -59,7 +59,104 @@
     @NotNull
     public List<ProjectUser> projects;
 
+    public User() {
+
+    }
+
+    public User(Long id, String firstName, String lastName, String email, String picture) {
+        this.id = id;
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.email = email;
+        this.picture = picture;
+    }
+
+
     public String getFullName() {
         return firstName + " " + lastName;
     }
+
+    public Long getId() {
+        return id;
+    }
+
+    public Organization getOrganization() {
+        return organization;
+    }
+
+    public String getFirstName() {
+        return firstName;
+    }
+
+    public String getLastName() {
+        return lastName;
+    }
+
+    public String getEmail() {
+        return email;
+    }
+
+    public String getPicture() {
+        return picture;
+    }
+
+    public List<Profile> getProfiles() {
+        return profiles;
+    }
+
+    public List<ProjectUser> getProjects() {
+        return projects;
+    }
+
+    public static final class Attendee {
```

Comment:
I prefer to keep Attendee into the Response or Request object, and not as part of the model, as it's a Data transfer object (and not something we really persist)